lib/mumukit/templates/multi_file_hook.rb

```diff
@@ -0,0 +1,25 @@
+module Mumukit
+  class Templates::MultiFileHook < Templates::FileHook
+    include Mumukit::Templates::WithMultipleFiles
+
+    def compile(request)
+      return super(request) unless has_files?(request)
+
+      self.request = request
+      compile_file_content(request).map { |filename, content| write_tempfile!(content, filename) }
```

Comment:
I have to sanitize the filename

Reply:
`filename` comes from users' direct input, so I meant checking that it doesn't have "/", "../", ...

Reply:
Oh, i see, you are right!